compiler/haskell-ide-core/src/Development/IDE/UtilGHC.hs

```diff
@@ -62,25 +63,68 @@ runGhcEnv env act = do
         cleanTempFiles dflags
         cleanTempDirs dflags
 
-
--- Fake DynFlags which are mostly undefined, but define enough to do a little bit
+-- Fake DynFlags which are mostly undefined, but define enough to do a
+-- little bit.
 fakeDynFlags :: DynFlags
 fakeDynFlags = defaultDynFlags settings ([], [])
     where
         settings = Settings
-            {sTargetPlatform = Platform
-                {platformWordSize = 8
-                ,platformOS = OSUnknown
-                ,platformUnregisterised = True
-                }
-            ,sPlatformConstants = PlatformConstants
-                {pc_DYNAMIC_BY_DEFAULT = False
-                ,pc_WORD_SIZE = 8
-                }
+                   { sGhcNameVersion=ghcNameVersion
+                   , sFileSettings=fileSettings
+                   , sTargetPlatform=platform
+                   , sPlatformMisc=platformMisc
+                   , sPlatformConstants=platformConstants
+                   , sToolSettings=toolSettings
+                   }
+        fileSettings = FileSettings {
+          -- fileSettings_tmpDir="."
+          }
+        toolSettings = ToolSettings {
+          toolSettings_opt_P_fingerprint=fingerprint0
+          }
+        ghcNameVersion =
+          GhcNameVersion{
+          ghcNameVersion_programName="ghc"
+          , ghcNameVersion_projectVersion=cProjectVersion
+          }
+        platformMisc = PlatformMisc {
 #ifndef GHC_STABLE
-            ,sIntegerLibraryType = IntegerSimple
+          platformMisc_integerLibraryType=IntegerSimple
 #endif
-            ,sProjectVersion = cProjectVersion
-            ,sProgramName = "ghc"
-            ,sOpt_P_fingerprint = fingerprint0
-            }
+          }
+        platform =
+          Platform{
+            platformWordSize=8
+          , platformOS=OSUnknown
+          , platformUnregisterised=True
+        }
+        platformConstants =
+          PlatformConstants {
+            pc_DYNAMIC_BY_DEFAULT=False
+          , pc_WORD_SIZE=8
+          -- , pc_STD_HDR_SIZE=1
```

Comment:
Can we just delete these rather than commenting out

Reply:
Oops!

Reply:
✔️